tests/test_dataset.py

```diff
@@ -14,23 +15,47 @@
 
 node_feats = [Nfeat.RESTYPE, Nfeat.POLARITY, Nfeat.BSA, Nfeat.RESDEPTH, Nfeat.HSE, Nfeat.INFOCONTENT, Nfeat.PSSM]
 
+def _cal_mean_std( # noqa: MC0001, pylint: disable=too-many-locals
+                       hdf5_path: str,
+                       features_transform:dict,
+                       feat:str
+    ):
+
+    with h5py.File(hdf5_path, 'r') as f:
+        entry_names = [entry for entry, _ in f.items()]
+
+        df_dict = {}
+        df_dict['id'] = entry_names
+
+        transform = False
+        transform = features_transform.get(feat, {}).get('transform')
+                
+        df_dict[feat] = [
+            f[entry_name][Nfeat.NODE][feat][:]
+            if f[entry_name][Nfeat.NODE][feat][()].ndim == 1
+            else f[entry_name][Nfeat.NODE][feat][()] for entry_name in entry_names]
+        #apply transformation
+        if transform:
+            df_dict[feat]=[transform(row) for row in df_dict[feat]]
+        
+        df = pd.DataFrame(data=df_dict)
+ 
+        means = {col: round(np.concatenate(df[col].values).mean(), 1) if isinstance(df[col].values[0], np.ndarray) \
```

Comment:
means is actually a single mean here, so I suggest to change the name to `mean`. Same for devs below, change it to `dev`. You'll have something like:
```python
round(np.concatenate(feat_values.mean(), 1) if isinstance(feat_values[0], np.ndarray) else round(np.array(feat_values).mean(), 1)
```